test/system.jl

```diff
@@ -463,6 +463,10 @@ ps = PathSet(; symlink=true)
                     content = read(joinpath(docsdir, "src", "api.md"), String)
                     @test write(p"writefp", content) == write("writestr", content)
                 end
+
+                @testset "Mmap.mmap" begin
+                    @test Mmap.mmap(p"README.md") == Mmap.mmap("README.md")
```

Comment:
I believe this needs to be finalized before the directory gets cleaned up. If you don't want to test Mmap semantics in this package you could mmap a temp file that outlives the tests on CI.

Reply:
So something like?

```
fpmap, strmap = Mmap.mmap(p"README.md"), Mmap.mmap("README.md")
@test fpmap == strmap
finalize.([fpmap, strmap])
```

Reply:
Yes. Now I'm not super sure whether it's okay to manually call the finalizers but we'll see when CI either works or doesn't.

Reply:
It seems to work on the REPL as long as you don't reference the variable after.

Reply:
Did you test on Windows too?

Reply:
Lol, that's what CI is for...

Reply:
Looks like that worked. Thanks for catching that @iamed2 !

Reply:
yw! I only knew because I remembered the conversation you had with Jacob Quinn about mmap and Windows.